Run Information
========================
cognizant.intelligent.test.scripter.engine : 1.4
java.runtime.name    : OpenJDK Runtime Environment
java.version         : 11.0.2
java.home            : /usr/lib/jvm/java-11-openjdk-amd64
os.name              : Linux
os.arch              : amd64
os.version           : 4.15.0-50-generic
file.encoding        : UTF-8
========================
Run Started on Wed Dec 11 10:56:36 UTC 2019
Loading Browser Profile
SLF4J: Class path contains multiple SLF4J bindings.
SLF4J: Found binding in [jar:file:/drone/src/lib/slf4j-nop-1.7.28.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: Found binding in [jar:file:/drone/src/lib/slf4j-simple-1.7.10.jar!/org/slf4j/impl/StaticLoggerBinder.class]
SLF4J: See http://www.slf4j.org/codes.html#multiple_bindings for an explanation.
SLF4J: Actual binding is of type [org.slf4j.helpers.NOPLoggerFactory]
Loading RunManager
[Scenario:BasicFeatures] [TestCase: DemoTestCase] [Description: DemoTestCase] [Browser: Chrome] [BrowserVersion: Default] [Platform: ANY][ExistingBrowser: false]
----------------------------------------------------------
[1] TestCase selected for execution from [//cits/BasicWebFlow_Environment1]
----------------------------------------------------------
Initializing Report
ApiLink not available!!
Run Manager true
Continue Execution true
Shutting Down Thread as there is no need for SelectiveThreading
Inside Task Run
Running Iteration 1
Launching Chrome
Launching Remote Driver
2019-12-11 10:56:36:12S +0000 [INFO] org.openqa.selenium.remote.DesiredCapabilities chrome:Using `new ChromeOptions()` is preferred to `DesiredCapabilities.chrome()`
2019-12-11 10:56:37:12S +0000 [SEVERE] com.cognizant.cognizantits.engine.core.Task onError:Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'a31ac1d75bfb', ip: '192.168.64.3', os.name: 'Linux', os.arch: 'amd64', os.version: '4.15.0-50-generic', java.version: '11.0.2'
Driver info: driver.version: SeleniumDriver
org.openqa.selenium.remote.UnreachableBrowserException: Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'a31ac1d75bfb', ip: '192.168.64.3', os.name: 'Linux', os.arch: 'amd64', os.version: '4.15.0-50-generic', java.version: '11.0.2'
Driver info: driver.version: SeleniumDriver
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:573)
	at org.openqa.selenium.remote.RemoteWebDriver.startSession(RemoteWebDriver.java:213)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:131)
	at org.openqa.selenium.remote.RemoteWebDriver.<init>(RemoteWebDriver.java:144)
	at com.cognizant.cognizantits.engine.drivers.WebDriverFactory.createRemoteDriver(WebDriverFactory.java:414)
	at com.cognizant.cognizantits.engine.drivers.WebDriverFactory.create(WebDriverFactory.java:258)
	at com.cognizant.cognizantits.engine.drivers.WebDriverFactory.create(WebDriverFactory.java:171)
	at com.cognizant.cognizantits.engine.drivers.WebDriverFactory.createRemote(WebDriverFactory.java:157)
	at com.cognizant.cognizantits.engine.drivers.SeleniumDriver.launchDriver(SeleniumDriver.java:63)
	at com.cognizant.cognizantits.engine.core.Task.launchBrowser(Task.java:158)
	at com.cognizant.cognizantits.engine.core.Task.runIteration(Task.java:128)
	at com.cognizant.cognizantits.engine.core.Task.run(Task.java:85)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)
Caused by: java.net.ConnectException: Failed to connect to hub/192.168.64.4:4444
	at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:247)
	at okhttp3.internal.connection.RealConnection.connect(RealConnection.java:165)
	at okhttp3.internal.connection.StreamAllocation.findConnection(StreamAllocation.java:257)
	at okhttp3.internal.connection.StreamAllocation.findHealthyConnection(StreamAllocation.java:135)
	at okhttp3.internal.connection.StreamAllocation.newStream(StreamAllocation.java:114)
	at okhttp3.internal.connection.ConnectInterceptor.intercept(ConnectInterceptor.java:42)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
	at okhttp3.internal.cache.CacheInterceptor.intercept(CacheInterceptor.java:93)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
	at okhttp3.internal.http.BridgeInterceptor.intercept(BridgeInterceptor.java:93)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at okhttp3.internal.http.RetryAndFollowUpInterceptor.intercept(RetryAndFollowUpInterceptor.java:126)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:147)
	at okhttp3.internal.http.RealInterceptorChain.proceed(RealInterceptorChain.java:121)
	at okhttp3.RealCall.getResponseWithInterceptorChain(RealCall.java:200)
	at okhttp3.RealCall.execute(RealCall.java:77)
	at org.openqa.selenium.remote.internal.OkHttpClient.execute(OkHttpClient.java:103)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:105)
	at org.openqa.selenium.remote.ProtocolHandshake.createSession(ProtocolHandshake.java:74)
	at org.openqa.selenium.remote.HttpCommandExecutor.execute(HttpCommandExecutor.java:136)
	at org.openqa.selenium.remote.RemoteWebDriver.execute(RemoteWebDriver.java:552)
	... 14 more
Caused by: java.net.ConnectException: Connection refused (Connection refused)
	at java.base/java.net.PlainSocketImpl.socketConnect(Native Method)
	at java.base/java.net.AbstractPlainSocketImpl.doConnect(AbstractPlainSocketImpl.java:399)
	at java.base/java.net.AbstractPlainSocketImpl.connectToAddress(AbstractPlainSocketImpl.java:242)
	at java.base/java.net.AbstractPlainSocketImpl.connect(AbstractPlainSocketImpl.java:224)
	at java.base/java.net.SocksSocketImpl.connect(SocksSocketImpl.java:403)
	at java.base/java.net.Socket.connect(Socket.java:591)
	at okhttp3.internal.platform.Platform.connectSocket(Platform.java:129)
	at okhttp3.internal.connection.RealConnection.connectSocket(RealConnection.java:245)
	... 36 more

[DEBUG]   | Could not start a new session. Possible causes are invalid address of the remote server or browser start-up failure.
Build info: version: '3.141.59', revision: 'e82be7d358', time: '2018-11-14T08:17:03'
System info: host: 'a31ac1d75bfb', ip: '192.168.64.3', os.name: 'Linux', os.arch: 'amd64', os.version: '4.15.0-50-generic', java.version: '11.0.2'
Driver info: driver.version: SeleniumDriver

===================================================================================================

2019-12-11 10:56:37:12S +0000 [SEVERE] com.cognizant.cognizantits.engine.reporting.impl.html.HtmlTestCaseHandler updateTestLog:null
java.lang.NullPointerException
	at com.cognizant.cognizantits.engine.reporting.impl.html.HtmlTestCaseHandler.updateTestLog(HtmlTestCaseHandler.java:103)
	at com.cognizant.cognizantits.engine.reporting.TestCaseReport.updateTestLog(TestCaseReport.java:134)
	at com.cognizant.cognizantits.engine.reporting.TestCaseReport.updateTestLog(TestCaseReport.java:105)
	at com.cognizant.cognizantits.engine.core.Task.onError(Task.java:195)
	at com.cognizant.cognizantits.engine.core.Task.onError(Task.java:183)
	at com.cognizant.cognizantits.engine.core.Task.runIteration(Task.java:140)
	at com.cognizant.cognizantits.engine.core.Task.run(Task.java:85)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
	at java.base/java.lang.Thread.run(Thread.java:834)


---------------------------------------------------
Testcase Name        : BasicFeatures:DemoTestCase
Executed Steps       : 1
Passed Steps         : 0
Failed Steps         : 0
Time Taken           : 00:00:00
-----------------------------------------------------

Run Finished on Wed Dec 11 10:56:37 UTC 2019
-----------------------------------------------------
ExecutionDate        : 11-Dec-2019 10-56-36
Executed TestCases   : 1
Passed TestCases     : 0
Failed TestCases     : 1
Time Taken           : 00:00:00
ReportPath           : file:///Projects/DemoProject/Results/TestExecution/cits/BasicWebFlow_Environment1/11-Dec-2019 10-56-36/summary.html
-----------------------------------------------------
